.github/workflows/make_bundle_conda.yml

```diff
@@ -401,21 +406,22 @@ jobs:
         working-directory: napari-packaging
         run: python build_installers.py --location=../napari-source
 
-      - name: Collect licenses
-        id: licenses
-        shell: bash -el {0}
-        working-directory: napari-packaging
-        run: |
-          licenses_zip_path=$(python build_installers.py --licenses)
-          echo "LICENSES_ARTIFACT_PATH=$licenses_zip_path" >> $GITHUB_ENV
-          echo "LICENSES_ARTIFACT_NAME=$(basename ${licenses_zip_path})" >> $GITHUB_ENV
-          echo "::set-output name=licenses_artifact::$licenses_zip_path"
+      # - name: Collect licenses
+      #   id: licenses
+      #   shell: bash -el {0}
+      #   working-directory: napari-packaging
+      #   run: |
+      #     licenses_zip_path=$(python build_installers.py --licenses)
+      #     echo "LICENSES_ARTIFACT_PATH=$licenses_zip_path" >> $GITHUB_ENV
+      #     echo "LICENSES_ARTIFACT_NAME=$(basename ${licenses_zip_path})" >> $GITHUB_ENV
+      #     echo "licenses_artifact=${licenses_zip_path}" >> $GITHUB_OUTPUT
 
-      - name: Upload License Artifact
-        uses: actions/upload-artifact@v2
-        with:
-          path: ${{ env.LICENSES_ARTIFACT_PATH }}
-          name: ${{ env.LICENSES_ARTIFACT_NAME }}
+
+      # - name: Upload License Artifact
+      #   uses: actions/upload-artifact@v2
+      #   with:
+      #     path: ${{ env.LICENSES_ARTIFACT_PATH }}
+      #     name: ${{ env.LICENSES_ARTIFACT_NAME }}
```

Comment:
same here?

Reply:
In fact, I was not done and this shouldn't have been merged. The installers being generated now are generally broken in subtle ways til I fix some stuff on the constructor stack... not urgent or important enough to revert, but I'll address this during the week.

In general, if I have _not_ requested a review, this is not ready for review or merge, regardless the status of the PR. I'll open as draft from now on, but please double check before pulling the trigger.